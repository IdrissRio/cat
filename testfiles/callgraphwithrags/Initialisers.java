// PackageName: org.extendj.callgraph.Initialisers
// MethodName: main

package org.extendj.callgraph;
class A extends T {
  public A() { System.out.println("A"); }
}

class T {
  static { System.out.println("Initialiser"); }
  public T() { System.out.println("T"); }
}

class Initialisers {
  public static void main(String[] args) { A a = new A(); }
}