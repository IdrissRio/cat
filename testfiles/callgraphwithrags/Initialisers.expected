org.extendj.callgraph.Initialisers::main(String[])->org.extendj.callgraph.A::A()
org.extendj.callgraph.A::A()->org.extendj.callgraph.T::T(),java.io.PrintStream::println(java.lang.String)
org.extendj.callgraph.T::T()->java.lang.Object::<init>(),java.io.PrintStream::println(java.lang.String),org.extendj.callgraph.T::clinit<>
org.extendj.callgraph.T::clinit<>->java.io.PrintStream::println(java.lang.String)

